test/unit/chips/Chip.test.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import {assert} from 'chai';
+import {mount} from 'enzyme';
+import {Chip} from '../../../packages/chips';
+
+suite('Chip');
+
+test('classNames adds classes', () => {
+  const wrapper = mount(<Chip className='test-class-name' />);
+  assert.isTrue(wrapper.hasClass('test-class-name'));
+});
+
+test('renders chip', () => {
+  const wrapper = mount(<Chip>Hello world</Chip>);
+  assert.exists(wrapper.find('.mdc-chip'));
+});
+
+test('#adapter.addClass adds class to state.classList', () => {
+  const wrapper = mount(<Chip>Jane Doe</Chip>);
+  wrapper.instance().foundation_.adapter_.addClass('test-class-name');
```

Comment:
Should we be accessing private properties in a test? Is there a way to spy on `addClass`? I'm seeing this pattern here, in the next test, and one of the `ChipSet` tests.

Reply:
Changed the `ChipSet` test to use its public adapter getter, but I'm not sure if there's a way to do the same for `Chip` since it's an HOC wrapped in `withRipple`.

Reply:
Are you using Jest to run the tests? If so, there are some handy automatic mocking functions you can use and pass arbitrary props down to Chip and check its `className` property.

http://jestjs.io/docs/en/es6-class-mocks#automatic-mock

Reply:
We're not using Jest :( we're using Karma like in the MDC Web repo